src/vs/workbench/contrib/terminal/browser/terminalQuickFixBuiltinActions.ts

```diff
@@ -129,7 +129,7 @@ export function gitPushSetUpstream(): IInternalOptions {
 			lineMatcher: GitPushOutputRegex,
 			anchor: 'bottom',
 			offset: 0,
-			length: 5
+			length: 15
```

Comment:
This was meant to be _wrapped lines_ in my eyes, so we might want to clear that up later? I made a change in this area recently as it was causing a freeze

Reply:
yeah I think that's why this was confusing to both of us. we should make this clearer